_docs/reference/expression-language.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Mixer Config Expression Language
+overview: Mixer config expression language reference.
+
+order: 43
+
+layout: docs
+type: markdown
+---
+
+{% capture mixerConfig %}{{home}}/docs/concepts/policy-and-control/mixer-config.html{% endcapture %}
+
+This page describes how to use the Mixer config expression language (CEXL).
+
+## Background
+
+Mixer configuration uses an expression language (CEXL) to specify [selectors]({{mixerConfig}}#selectors) and [mapping expressions]({{mixerConfig}}#attribute-expressions). CEXL expressions map a set of typed [attributes]({{home}}/docs/concepts/policy-and-control/attributes.html) and constants to a typed [value](https://github.com/istio/api/blob/master/mixer/v1/config/descriptor/value_type.proto#L23).
+  
+```
+f(attr0, attr1, ..., constant0, constant1, ...) --> value
+```
+
+## Syntax
+
+CEXL accepts a subset of the **golang** expressions, which defines the syntax. CEXL parses a string input using the builtin [golang parser](https://golang.org/pkg/go/parser/#ParseExpr) to produce a simplified AST that consists of functions, variables and, constants. CEXL treats operators as functions and implements a subset of the golang operators that constrains the set of accepted expressions. CEXL supports arbitrary parenthesization. 
```

Comment:
How about replacing golang with Go, and making it a link to a relevant part of the Go manual?